bin/sendsketch_stainer.py

```diff
@@ -23,4 +23,8 @@
 					output= line.rstrip().split("\t")[1]
 	else:
 		output="No_stain"
-print(output)
+
+# print(output)
+
+with open("sendsketch_stainer_output.txt", "w") as f:
+    f.writelines(output)
```

Comment:
Since only one `stdout` is allowed per process and the `print(output)` of this file wasn't being used anywhere, I have saved it via a file.

At NF level, I rename this file using 
```
    mv sendsketch_stainer_output.txt ${pair_id}_sendsketch_stainer_output.txt

```

Reply:
This script isn't used anymore and can safely be removed. This was the earlier version of the sendsketch_to_prokka.py script which has been superseded by the new.

Reply:
Are you sure? 

I see that `sendsketch_stainer.py` is being used in `SCREEN...`. I'd like to confirm the status of `assess_mash_screen.py`

Reply:
The module in branch emilio only uses sendsketsh.sh and sendsketch_to_prokka.py and should not use sendsketch_stainer.py as this script was just a lesser version of the new sendsketch_to_prokka.py. 
Also the assess_mash_screen.py is redundant as the pipeline no longer uses mash.

Reply:
I've removed `sendsketch_to_prokka.py` from the codebase now.